pyriemann/estimation.py

```diff
@@ -638,11 +640,19 @@ def transform(self, X):
         return np.array(out)
 
 
+@deprecated(
+    "HankelCovariances is deprecated and will be removed in 0.8.0; "
+    "please use TimeDelayCovariances."
+)
 class HankelCovariances(BaseEstimator, TransformerMixin):
```

Comment:
I would say you need to inherit from TimeDelayCovariances as otherwise code that used to work will break. If I now import HankelCovariances in my code nothing works exept the instance creation without any parameter.